lib/stripe/stripe_client.rb

```diff
@@ -280,53 +280,47 @@ def handle_error_response(http_resp, context)
     def specific_api_error(resp, error_data, context)
       Util.log_error("Stripe API error",
                      status: resp.http_status,
-                     error_code: error_data["code"],
-                     error_message: error_data["message"],
-                     error_param: error_data["param"],
-                     error_type: error_data["type"],
+                     error_code: error_data[:code],
+                     error_message: error_data[:message],
+                     error_param: error_data[:param],
+                     error_type: error_data[:type],
```

Comment:
These should've been symbols (not strings) all along, but because the problem was duplicated in the tests, I never noticed because everything passed.

Reply:
Nice catch!